build_tools/python/e2e_test_framework/definitions/iree_definitions.py

```diff
@@ -324,7 +325,7 @@ def build(cls, module_generation_config: ModuleGenerationConfig,
             module_execution_config: ModuleExecutionConfig,
             target_device_spec: common_definitions.DeviceSpec,
             input_data: common_definitions.ModelInputData,
-            tool: E2EModelRunTool):
+            tool: E2EModelRunTool, tags: Sequence[str]):
```

Comment:
Maybe give it a default value?

```suggestion
            tool: E2EModelRunTool, tags: Optional[Sequence[str]] = None):
    tags = [] if tags is None else list(tags)
```

Reply:
Added